src/containers/AssetForm.js

```diff
@@ -1,16 +1,416 @@
+];
+
+const DATA_CATEGORY_LABELS = [
+  {value: 'education', label: "Education"},
+  {value: 'racial', label: "Racial or ethnic origin"},
+  {value: 'employment', label: "Employment"},
+  {value: 'political', label: "Political opinions"},
+  {value: 'financial', label: "Financial"},
+  {value: 'unions', label: "Trade union membership"},
+  {value: 'social', label: "Lifestyle and social circumstances"},
+  {value: 'religious', label: "Religious or similar beliefs"},
+  {value: 'visual', label: "Visual images"},
+  {value: 'health', label: "Physical or mental health details"},
+  {value: 'research', label: "Research data"},
+  {value: 'sexual', label: "Sexual life and orientation"},
+  {value: 'medical', label: "Medical records"},
+  {value: 'genetic', label: "Genetic information"},
+  {value: 'children', label: "Data about children 16"},
+  {value: 'biometric', label: "Biometric information"},
+  {value: 'criminal', label: "Criminal proceedings"},
+];
+
+const RISK_TYPE_LABELS = [
+  {value: 'financial', label: "The University would be exposed to financial loss by disclosing this information"},
+  {value: 'operational', label: "Damage or loss of this information would impact core day-to-day operations."},
+  {value: 'safety', label: "Staff or students would be exposed to personal danger if this information was disclosed."},
+  {value: 'compliance', label: "The University would be compliant with all the necessary laws and regulations related to holding this information."},
+  {value: 'reputational', label: "The University would receive significant negative publicity"},
+];
+
+const DIGITAL_STORAGE_SECURITY_LABELS = [
+  {value: 'pwd_controls', label: "Password controls"},
+  {value: 'acl', label: "Access control lists"},
+  {value: 'backup', label: "Backup"},
+  {value: 'encryption', label: "Encryption"},
+];
+
+const PAPER_STORAGE_SECURITY_LABELS = [
+  {value: 'locked_cabinet', label: "Locked filing cabinet"},
+  {value: 'safe', label: "Safe"},
+  {value: 'locked_room', label: "Locked room"},
+  {value: 'locked_building', label: "Locked building"},
+];
+
+const retentionStyle = {
+  borderWidth: '1px 1px 0',
+  borderStyle: 'solid',
+  borderColor: '#d1d1d1',
+  padding: '10px 20px'
+};
+
+const storageFormatGroupStyle = {
+  display: 'flex',
+  borderLeft: '1px solid #d1d1d1'
+};
+
+const storageFormatButtonStyle = {
+  width: 'auto',
+  borderWidth: '1px 1px 1px 0',
+  borderStyle: 'solid',
+  borderColor: '#d1d1d1',
+  padding: "10px 20px"
+};
 
 /*
   Renders the form for the creation/editing of an Asset.
   */
-const AssetForm = ({ match }) => (
-  <div className="App-main">
-    <AssetFormHeader title={match.url === '/asset/create' ? 'Create new asset' : 'Editing: Some asset'} />
-    <div>
-      <h1>Asset Creation/Editing Form</h1>
-    </div>
-  </div>
+class AssetForm extends Component {
+
+  constructor() {
+    super();
+
+    this.fetchMatchingUsers = _.debounce(this.fetchMatchingUsers, 200);
```

Comment:
We are truly in the future when switch debouncing makes it into the digital realm.